src/views/Operations/ServerPowerOperations/ServerPowerOperations.vue

```diff
@@ -272,17 +278,34 @@ export default {
           this.$bvModal
             .msgBoxConfirm(modalMessage, modalOptions)
             .then((confirmed) => {
-              if (confirmed) this.$store.dispatch('controls/serverSoftReboot');
+              if (confirmed) {
+                if (!this.isReadOnlyUser) {
+                  this.showHostConsole();
+                }
+                this.$store.dispatch('controls/serverSoftReboot');
+              }
             });
         } else if (this.form.rebootOption === 'immediate') {
           this.$bvModal
             .msgBoxConfirm(modalMessage, modalOptions)
             .then((confirmed) => {
-              if (confirmed) this.$store.dispatch('controls/serverHardReboot');
+              if (confirmed) {
+                if (!this.isReadOnlyUser) {
+                  this.showHostConsole();
+                }
+                this.$store.dispatch('controls/serverHardReboot');
+              }
             });
         }
       });
     },
+    showHostConsole() {
+      window.open(
+        '#/console/host-console-console',
+        '_blank',
+        'directories=no,titlebar=no,toolbar=no,location=no,status=no,menubar=no,scrollbars=no,resizable=yes,width=600,height=550'
```

Comment:
scrollbars=no why ? We can remove it as by default scrollbars are set to yes

Reply:
I guess the scroll bar is not required in the pop-up.. That is why the scrollbar=no.
It is same as we have implemented in the host-console page.